### DISCUSSION TITLE
custom NER on top of  en-core-web-sm  tagger and  parser

### DISCUSSION THREAD

**Discussion by myeghaneh** [explosion/spaCy#6764] [Help: Coding & Implementations]
I want to train my custom NER on top of the parser and tagger of en-core-web-sm , could you give me some hints on how can I do it to have the best result

I have tried here https://github.com/explosion/spaCy/discussions/6735 , still is not good enough 
many thanks

**Comment by svlandeg** [comment]
Hi @myeghaneh, please avoid opening new issues just to get your problem "bumped" in priority.

We're actively maintaining the issue tracker and the discussions board, but you need to give people proper time to respond. I will lock this thread as you've referred to another one which has more information.
